src/Nri/Ui/CharacterIcon/V2.elm

```diff
@@ -1159,30 +1161,34 @@ renderHeadshotFrame :
     , clippedSvg : Svg.Svg Never
     , secondaryPath : String
     , secondaryColor : Css.Color
+    , circleStyles : List Css.Style
+    , styles : List Css.Style
```

Comment:
we never use `styles` in this record any more. What if we were explicit about this by calling this `circleClipStyles` and then explicitly passing some empty lists? We got lucky that leaving the styles out of that element worked this time, but next time we're styling something it'll be weird for styles not to be applied to some arbitrary element.

If that doesn't sound good to you, could you add a note so we know where styles aren't being applied so someone could fix it up when they need it in the future?

Reply:
Actually I think we don't need those anymore, we just need the circle styles. I don't think we will need them.